src/redux/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import RocketSlice from './rockets/RocketSlice';
+import MissionSlice from './missions/MissionSlice';
+
+const store = configureStore({
+  reducer: {
+    rocket: RocketSlice,
+    mission: MissionSlice,
+  },
+});
+export default store;
```

Comment:
- [ ] However much both `configureStore` and `redux logger` are related to the state management library but they serve different purposes. The redux logger purpose has been well explained by Priscilla and configureStore simplifies the setup of the redux store by simply combining several pieces of configuration as used in your code. Since we need to stick to the project requirements, kindly consider using a `logger` from redux-logger as stated by previous reviewers following the guidance provided ☺️.